pkg/cmd/create/create_test.go

```diff
@@ -62,7 +62,7 @@ func AssertPreview(t *testing.T, customService string) {
 
 	serviceName := customService
 	if serviceName == "" {
-		serviceName = repo
+		serviceName = "default_service_name"
```

Comment:
Dont hardcode, use a variable `defaultService` similar to customService.

Reply:
Done : )